### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
- [ ] Did you write/update appropriate tests
- [ ] Release notes updated (if appropriate)
- [ ] Appropriate logging output
- [ ] Issue linked
- [ ] Docs updated (or issue created)

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/gh-gei/commit/c3430f3d3c55fcff07cdc19e9e2bd4e50bedd02e#commitcomment-79410304

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/gh-gei/commit/75ea0bcd6d88cadc474c7c3fe5026adbdc404995

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: releasenotes/v0.19.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- Added `inventory-report` command to `ado2gh`. This command will generate a few CSV reports of all the orgs, team projects, repos and pipelines it can access. Along with some extra information that can be useful when planning a large migration (e.g. PR count per repo, most active contributer, pushes in past year, last push date, etc)
+- Added `inventory-report` command to `ado2gh`. This command will generate a few CSV reports of all the orgs, team projects, repos and pipelines it can access. Along with some extra information that can be useful when planning a large migration (e.g. PR count per repo, most active contributor, pushes in past year, last push date, etc)
```
Comment (jsoref) [inline releasenotes/v0.19.md:1]: Some projects don't like changing release notes. Happy to drop such changes.

File: releasenotes/v0.7.md [hunk 2]
```diff
@@ -4,5 +4,5 @@
 - Show a generic error message instead of the actual one for unhandled exceptions in non-verbose mode.
 - Exit code is now 1 instead of 0 in case of an error.
 - Errors are written to std error instead of std out. 
-- Adding Support to get multi page results from Github API.
-- The Github to Github migrations are no longer limited to 30 repos. 
+- Adding Support to get multi page results from GitHub API.
+- The GitHub to GitHub migrations are no longer limited to 30 repos.
```
Comment (jsoref) [inline releasenotes/v0.7.md:8]: brand

File: src/OctoshiftCLI.Tests/AdoApiTests.cs [hunk 3]
```diff
@@ -1178,7 +1178,7 @@ public async Task LockRepo_Should_Send_Correct_Payload()
         [Fact]
         public async Task IsCallerOrgAdmin_Returns_True_When_Caller_Is_Org_Admin()
         {
-            // Arrnage
+            // Arrange
```
Comment (jsoref) [inline src/OctoshiftCLI.Tests/AdoApiTests.cs:1181]: ?

File: src/OctoshiftCLI.Tests/GithubApiTests.cs [hunk 4]
```diff
@@ -990,7 +990,7 @@ public async Task GetTeamSlug_Returns_The_Team_Slug()
                     description = "A great team.",
                     privacy = "closed",
                     permission = "admin",
-                    members_url = "https://api.github.com/teams/1/members/membber",
+                    members_url = "https://api.github.com/teams/1/members/member",
```
Comment (jsoref) [inline src/OctoshiftCLI.Tests/GithubApiTests.cs:993]: If this is intentional, might I suggest:

```suggestion
                    members_url = "https://api.github.com/teams/1/members/member_",
```

File: src/OctoshiftCLI.Tests/GithubApiTests.cs [hunk 5]
```diff
@@ -1665,7 +1665,7 @@ public async Task RepoExists_Should_Return_False_If_Repo_Does_Not_Exist_301()
         }
 
         [Fact]
-        public async Task RepoExists_Throws_When_Underlying_HtttRepsponseException_Status_Is_Not_NotFound_Or_Moved()
+        public async Task RepoExists_Throws_When_Underlying_HtttResponseException_Status_Is_Not_NotFound_Or_Moved()
```
Comment (jsoref) [inline src/OctoshiftCLI.Tests/GithubApiTests.cs:1668]: I'm assuming that these functions don't care about their names.
```diff
-HtttRepsponse
+HtttResponse
```

(I also suspect that `Httt` should be `Http`...)

File: src/OctoshiftCLI.Tests/Services/ReclaimServiceTests.cs [hunk 6]
```diff
@@ -745,7 +745,7 @@ public async Task ReclaimMannequin_AlreadyMapped_Force_Reclaim()
         }
 
         [Fact]
-        public async Task ReclaimMannequin_FailtToReclaim_LogsError_Throws_OctoshiftCliException()
+        public async Task ReclaimMannequin_FailedToReclaim_LogsError_Throws_OctoshiftCliException()
```
Comment (jsoref) [inline src/OctoshiftCLI.Tests/Services/ReclaimServiceTests.cs:748]:
```diff
-Failt
+Failed
```

It's possible this meant to be 
```suggestion
        public async Task ReclaimMannequin_FaultToReclaim_LogsError_Throws_OctoshiftCliException()
```

But that didn't seem likely from context...

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine.

File: src/OctoshiftCLI.Tests/GithubApiTests.cs (hunk 5 above)
Comment (dylan-smith) [inline src/OctoshiftCLI.Tests/GithubApiTests.cs:1668]:
```suggestion
        public async Task RepoExists_Throws_When_Underlying_HttpResponseException_Status_Is_Not_NotFound_Or_Moved()
```

**Comment by dylan-smith** [comment]
It all looks good except the changes to historical release notes. Can you remove those changes please and I can merge it in.

File: src/OctoshiftCLI.Tests/GithubApiTests.cs (hunk 5 above)
Comment (jsoref) [reply src/OctoshiftCLI.Tests/GithubApiTests.cs:1668]: Oops. lemme fix this :-)

**Review by dylan-smith — APPROVED** [review]

**Pull request merged**